cli/src/config.rs

```diff
@@ -42,6 +42,8 @@ pub struct Config {
     pub excluded_rules: Vec<RuleViolationKind>,
     #[serde(default)]
     pub pg_version: Option<Version>,
+    #[serde(default)]
+    pub assume_in_transaction: Option<bool>,
```

Comment:
I picked `assume_transaction` initially, but saw `--assume-in-transaction` as a suggestion in #106. That sounded better to me so I switched to it.

It's relatively easy to rename this, so happy to hear an argument for something else.